lte/protos/oai/mme_nas_state.proto

```diff
@@ -150,6 +150,10 @@ message UeContext {
   google.protobuf.Timestamp time_paging_response_timer_started = 209;
   uint32 paging_retx_count = 210;
   google.protobuf.Timestamp time_ics_rsp_timer_started = 211;
+
+  // 5G protos
+  uint32 amf_ue_ngap_id = 221;
+  uint32 ue_context_request = 222;
```

Comment:
can we rename this to be more explicit on what this field represents? if there is no such name, add a comment on top of the field